sklearn/externals/joblib/format_stack.py

```diff
@@ -264,7 +263,11 @@ def tokeneater(token_type, token, start, end, line):
         # dotted names
         tokeneater.name_cont = False
 
+            
+           
+
```

Comment:
actually please ignore this apparently this commit was not supposed to be included in the PR.